<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tenantwindow</class>
 <widget class="QMainWindow" name="tenantwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tw_Home">
     <attribute name="title">
      <string>首页</string>
     </attribute>
     <widget class="QTableView" name="Home_view">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>350</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="Home_Gourp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>551</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>操作</string>
      </property>
      <widget class="QLineEdit" name="Home_SearchLineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;关键字搜索&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Home_SearchButton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>81</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>注：查询关键字为空时，默认查看所有房屋信息</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>只查询可出租的房屋</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Home_myhomeButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>410</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>我的房子</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Home_CallButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>370</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>联系房东</string>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>450</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tw_Mine">
     <attribute name="title">
      <string>个人信息</string>
     </attribute>
     <widget class="QGroupBox" name="Mine_Group">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>591</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>操作</string>
      </property>
      <widget class="QPushButton" name="Mine_uiButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>修改个人信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Mine_upButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Mine_refreshButton">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;账号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;邮箱：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;手机：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;年龄：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>200</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;城市：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_account">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_email">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_age">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_city">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_Tel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mine_name">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Mine_quitButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>380</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
